src/main/java/org/dita/dost/util/DITAOTCopy.java

```diff
@@ -89,7 +89,11 @@ public void execute() throws BuildException {
             throw new BuildException("Destination directory not defined");
         }
         if (!destDir.exists()) {
-            throw new BuildException("Destination directory " + destDir + " does not exists");
+            try {
+                destDir.mkdir();
```

Comment:
👍  only did `SecurityException` because the pop-up help said that's what `.mkdir` throws :D but yes that makes sense.